### PR TITLE
Blog post for changing database starting id

### PR DESCRIPTION
**Pull request by JovicaSusa**

### CODE REVIEW

File: source/_posts/2015-06-01-changing-database-starting-id.markdown [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+```
+rails g task namespace task_name
+```
+This will create your task in lib/tasks with chosen namespace and task name.
+
+Here is our task and an explanation that follows.
+
+```
+namespace :database do
+  desc "Detect database that's being used and than increment its id"
```
Comment (basti) [inline source/_posts/2015-06-01-changing-database-starting-id.markdown]: @JovicaSusa spelling: "than increment its id" => "then increment its id".

**Comment by basti** [comment]
@JovicaSusa getting good at this. :smile: :+1:

One suggestion... think about including few reference links for each database autoincrement thing, like a link to StackOverflow where you picked up one of the solutions. Maybe just add them at the end, so they don't pollute your text.

**Comment by JovicaSusa** [comment]
@basti Thanks, I added link to StackOverflow and corrected spelling.

**Comment by codalicious** [comment]
@JovicaSusa :+1:

**Pull request merged**